app/modules/conversations/conversation/conversation_service.py

```
@@ -597,6 +693,103 @@ async def _generate_and_stream_ai_response(
+    ) -> Optional[Dict[str, Dict[str, Union[str, int]]]]:
+        """Get images from the most recent human message in the conversation"""
+        try:
+            # Get the most recent human message with attachments
+            latest_human_message = (
+                self.sql_db.query(Message)
+                .filter_by(
+                    conversation_id=conversation_id,
+                    type=MessageType.HUMAN,
+                    status=MessageStatus.ACTIVE,
+                )
+                .filter(Message.has_attachments == True)
```

Flake8 (7.2.0)

[error] comparison to True should be 'if cond is True:' or 'if cond:'

(E712)